internal/config/helpers.go

```diff
@@ -80,5 +82,5 @@ func DumpConfig(logger zerolog.Logger) {
 	configCopy.Azure.ClientSecret = replacement
 	configCopy.GCP.JSON = replacement
 	configCopy.Unleash.Token = replacement
-	logger.Info().Msgf("Configuration: %+v", configCopy)
+	logger.Info().Str("binary", path.Base(os.Args[0])).Msgf("Configuration: %+v", configCopy)
 }
```

Comment:
I want the binary name to be reported in logs because that is our cloudwatch stream name.